docs/install.md

```diff
@@ -199,6 +199,21 @@ By default the Dashboard accesses resources and performs actions in the cluster
 
 Typically when configuring impersonation you would have the proxy forward its ServiceAccount token in the `Authorization` header, and details of the user and groups in the `Impersonate-User` and `Impersonate-Group` headers respectively. See the docs of your chosen solution for details.
 
+When using a reverse proxy, with impersonation headers or the user's account, you should remove the dashboard's privileges to better maintain a "least privileged" approach.  This will make it less likely that the dashboard's `ServiceAccount` will be abused:
+
+```
+kubectl delete clusterrole tekton-dashboard-backend
+kubectl delete clusterrole tekton-dashboard-dashboard
+kubectl delete clusterrole tekton-dashboard-pipelines
+kubectl delete clusterrole tekton-dashboard-tenant
+kubectl delete clusterrole tekton-dashboard-triggers
+kubectl delete clusterrolebinding tekton-dashboard-backend
+kubectl delete rolebinding tekton-dashboard-pipelines -n tekton-pipelines
+kubectl delete rolebinding tekton-dashboard-dashboard -n tekton-pipelines
+kubectl delete rolebinding tekton-dashboard-triggers -n tekton-pipelines
+kubectl delete clusterrolebinding tekton-dashboard-tenant
```

Comment:
Maybe we should consider adding a label to these resources to more easily target them for deletion and avoid having to keep this list up to date. What do you think?

I think we'll likely add an additional manifest (or at least an installer script flag) at some point to install the Dashboard without these resources. This would probably make sense as part of the remaining work to improve the experience with custom RBAC, impersonation, etc.